inst/tests/other.Rraw

```diff
@@ -36,17 +36,17 @@ if (all(c("package:reshape","package:reshape2") %in% search())) {
 
 if (loaded[["ggplot2"]]) {
   DT = data.table( a=1:5, b=11:50, d=c("A","B","C","D"), f=1:5, grp=1:5 )
-  test(1.1, names(print(ggplot(DT,aes(b,f))+geom_point()))[c(1,3)], c("data","plot"))
+  test(1.1, names(print(ggplot(DT,aes(b,f))+geom_point()))[c(1,3)], c("data","scales")) # update as described in #3047
   test(1.2, DT[,print(ggplot(.SD,aes(b,f))+geom_point()),by=list(grp%%2L)],data.table(grp=integer()))  # %%2 to reduce time needed for ggplot2 to plot
   if (loaded[["hexbin"]]) {
     # Test reported by C Neff on 11 Oct 2011
-    test(1.3, names(print(ggplot(DT) + geom_hex(aes(b, f)) + facet_wrap(~grp)))[c(1,3)], c("data","plot"))
+    test(1.3, names(print(ggplot(DT) + geom_hex(aes(b, f)) + facet_wrap(~grp)))[c(1,3)], c("data","scales"))
   }
   # Test plotting ITime with ggplot2 which seems to require an as.data.frame method for ITime, #1713
   datetimes = c("2011 NOV18 09:29:16", "2011 NOV18 10:42:40", "2011 NOV18 23:47:12",
                 "2011 NOV19 01:06:01", "2011 NOV19 11:35:34", "2011 NOV19 11:51:09")
   DT = IDateTime(strptime(datetimes,"%Y %b%d %H:%M:%S"))
-  test(1.4, print(DT[,qplot(idate,itime)])$ranges, error="geom_point requires the following missing aesthetics: y")  # y=print(qplot(DT$idate,DT$itime))$ranges
+  test(1.4, tryCatch(print(DT[,qplot(idate,itime)])$ranges, message=function(m) m[["message"]]), "Don't know how to automatically pick scale for object of type ITime. Defaulting to continuous.\n") # updated as breaking change introduced to ggplot2, see #3047
   test(1.5, print(DT[,qplot(idate,as.POSIXct(itime,tzone=""))])$ranges, print(qplot(idate,as.POSIXct(itime,tzone=""),data=DT))$ranges)
```

Comment:
we don't have a message argument to test?

Reply:
Never really used `message()` until recently.  Good idea to add `message=` to `test()`.  `capture.output()` can capture output or message stream, but not both at the same time I discovered recently.  What springs to mind first is that `test()` could accept `output=` or `message=` and it could do it using `capture.output()` rather than `tryCatch`.

But why are we testing that ggplot returns this message anyway?  If they improve ggplot and the message goes away,  that'll be a breaking change for others.Rraw again.  But is that something we want to break on?  Same goes for why we were testing for the error.  I did that I know,  and I'm asking myself.  Maybe Jan thought I must have had a reason and followed my (bad) lead.

Reply:
I agree it is better to remove those tests are the risk of breaking in future in high